### DISCUSSION TITLE
Version 3.70 & Win/Mac一键安装包

### DISCUSSION THREAD

**Discussion by binary-husky** [binary-husky/gpt_academic#1491] [Announcements]
1. 支持绘制脑图，重构前端UI和Gradio依赖
2. 支持智谱的GLM4
3. 支持直接拖拽文件到上传区
4. 支持将图片粘贴到输入区
5. 修复若干隐蔽的内存BUG
6. 修复多用户冲突问题
7. 接入Deepseek Coder
8. AutoGen多智能体插件测试版
9. 修复本地模型在Windows下的加载BUG
10. 支持文心一言v4和星火v3

**注意：一键安装脚本OneKeyInstallerForWindowsAndMacOS.zip不支持中文路径/带空格的文件夹**

<hr /><em>This discussion was created from the release <a href='https://github.com/binary-husky/gpt_academic/releases/tag/'>Version 3.70 & Win/Mac一键安装包</a>.</em>

**Comment by DojHe** [comment]
怎样才能保存自定义按钮，并在下一次打开时自动加载。
在“界面外观”中勾选“输入清除键”后，下次打开还需要重新勾选，请问配置文件中，哪里可以直接修改？
谢谢！

**Comment by ahyzzzz** [comment]
Downloading Miniconda from https://repo.anaconda.com/miniconda/Miniconda3-py310_23.1.0-1-Windows-x86_64.exe to C:\OneKeyInstallerForWindowsAndMacOS\installer_files\miniconda_installer.exe
子目录或文件 C:\OneKeyInstallerForWindowsAndMacOS\installer_files 已经存在。
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (35) schannel: next InitializeSecurityContext failed: SEC_E_INVALID_TOKEN (0x80090308) - 给函数提供的标志无效

Miniconda failed to download.
请按任意键继续. . .


想问下这是怎么回事

**Comment by verigle** [comment]
文件解压后乱码如何解决？

**Comment by APLMY** [comment]
进程完成后就没反应了是怎么回事？
